tss-esapi/tests/integration_tests/structures_tests/buffers_tests/private.rs

```diff
@@ -0,0 +1,14 @@
+// Copyright 2023 Contributors to the Parsec project.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::convert::TryFrom;
+use tss_esapi::structures::Private;
+
+#[test]
+fn marshall_unmarshall() {
```

Comment:
I created to distinct test methods. Is this what you meant?
Initially I created one method, since it was what I found before. https://github.com/parallaxsecond/rust-tss-esapi/blob/main/tss-esapi/tests/integration_tests/interface_types_tests/structure_tags_tests.rs#L113

Reply:
Not really what I was thinking. I mean where an offset is actually used. Like how it is done for [Public](https://github.com/parallaxsecond/rust-tss-esapi/blob/main/tss-esapi/tests/integration_tests/structures_tests/buffers_tests/public.rs) I think you could be able to use this in test [common](https://github.com/parallaxsecond/rust-tss-esapi/blob/main/tss-esapi/tests/integration_tests/common/marshall.rs).

Reply:
Sorry for the confusion I do not know what I was reading but this is fine.